v2/cf-nodejs-logging-support/src/test/config-test.json

```diff
@@ -0,0 +1,40 @@
+{
+    "fields": [
+        {
+            "name": "logger",
+            "mandatory": true,
+            "source": {
+                "type": "static",
+                "value": "TEST"
+            },
+            "output": [
+                "msg-log"
+            ]
+        },
+        {
+            "name": "deactivated_field",
+            "mandatory": false,
+            "source": {
+                "type": "self",
+                "name": "component_instance"
+            },
+            "output": [
+                "msg-log"
+            ],
+            "deactivated": true
```

Comment:
Good suggestion. I would still go for `"disable": true`, it sounds more logical to me since a user would naturally think "I want to disable a field" instead of "I want to not enable a field". Do you agree?

Reply:
Yes, that's fine 👍